src/Controls/src/Core/Platform/Android/Extensions/ToolbarExtensions.cs

```diff
@@ -375,6 +375,9 @@ internal static void UpdateMenuItemIcon(this IMauiContext mauiContext, IMenuItem
 						}
 
 						menuItem.SetIcon(iconDrawable);
+
+						// Setting the title, so that tooltip text is displayed in native color
+						menuItem.SetTitle(toolBarItem?.Text ?? string.Empty);
```

Comment:
The change looks good, but I'm going to talk to the team to find out if we want this behavior change or delegate it to the system color.

Reply:
Maybe, include it in a **Platform Specific** to allow to choose between the two different behaviors, could be a good idea.

Reply:
I think if just testting the text is changing the color, then this may be a bug or feature in android...